### ISSUE TITLE
New video is not updated

### ISSUE THREAD

**Issue by Plushb9rry** [frog23/TomScottMap-updater#1]
a new video appeared on the channel around 23 hours ago, it does not seem to be updated, something is wrong here

_New Video Link - https://www.youtube.com/watch?v=iQGl-ffVtaM
Video ID - iQGl-ffVtaM_

_Either issue is on this repo or in TomScottMap_

**Comment by frog23** [comment]
Fixed now. When I cleaned up the code last week to upload it here, I moved it to a new folder but forgot to update some of the git settings, so the update works automatically. So the script calls would always hang on the username/password input. 
Thanks for pointing this out. I also noticed it yesterday evening, but haven't had time until now to look what was wrong.

**Comment by Plushb9rry** [comment]
I see, cool. it's been more than 3 years since(I guess) this project has started, I love to see your dedication to this project this far, as I do my 2nd-year b.tech CE at university.

**Comment by frog23** [comment]
Actually last Friday was the 5 year anniversary of my first commit on this project. A while ago I only used to update the map once every 4-6 months, but I tried to make it more frequent (at least once a month) in the past 1 1/2 years. Two months ago I build the automated updater, which takes out some of the boring work and also makes it easier for others to contribute. So for it has been working quite well, until I started cleaning it up last week. Anyways, so in the last months the map has been quite up to date.
